diff --git a/cypress/e2e/po/chat.po.ts b/cypress/e2e/po/chat.po.ts
index 64baf43..c322a27 100644
--- a/cypress/e2e/po/chat.po.ts
+++ b/cypress/e2e/po/chat.po.ts
@@ -64,14 +64,14 @@ export default class ChatPo extends ComponentPo {
   openViaKeyboard() {
     const isMac = Cypress.platform === 'darwin';
 
-    cy.get('body').type(isMac ? '{meta}{shift}k' : '{alt}k');
+    cy.get('body').click().type(isMac ? '{meta}{shift}k' : '{alt}k');
     this.isOpen();
   }
 
   closeViaKeyboard() {
     const isMac = Cypress.platform === 'darwin';
 
-    cy.get('body').type(isMac ? '{meta}{shift}k' : '{alt}k');
+    cy.get('body').click().type(isMac ? '{meta}{shift}k' : '{alt}k');
     this.isClosed();
   }
 
