diff --git a/cypress/e2e/po/chat.po.ts b/cypress/e2e/po/chat.po.ts
index 4b9dba6..64baf43 100644
--- a/cypress/e2e/po/chat.po.ts
+++ b/cypress/e2e/po/chat.po.ts
@@ -64,22 +64,14 @@ export default class ChatPo extends ComponentPo {
   openViaKeyboard() {
     const isMac = Cypress.platform === 'darwin';
 
-    if (isMac) {
-      cy.get('body').trigger('keydown', { metaKey: true, shiftKey: true, key: 'k', keyCode: 75 });
-    } else {
-      cy.get('body').trigger('keydown', { altKey: true, key: 'k', keyCode: 75 });
-    }
+    cy.get('body').type(isMac ? '{meta}{shift}k' : '{alt}k');
     this.isOpen();
   }
 
   closeViaKeyboard() {
     const isMac = Cypress.platform === 'darwin';
 
-    if (isMac) {
-      cy.get('body').trigger('keydown', { metaKey: true, shiftKey: true, key: 'k', keyCode: 75 });
-    } else {
-      cy.get('body').trigger('keydown', { altKey: true, key: 'k', keyCode: 75 });
-    }
+    cy.get('body').type(isMac ? '{meta}{shift}k' : '{alt}k');
     this.isClosed();
   }
 
