diff --git a/cypress/e2e/tests/features/console-input.spec.ts b/cypress/e2e/tests/features/console-input.spec.ts
index 3900cef..a6c4973 100644
--- a/cypress/e2e/tests/features/console-input.spec.ts
+++ b/cypress/e2e/tests/features/console-input.spec.ts
@@ -1,3 +1,4 @@
+import HomePagePo from '@rancher/cypress/e2e/po/pages/home.po';
 import ChatPo from '@/cypress/e2e/po/chat.po';
 import { ConsolePo } from '@/cypress/e2e/po/console.po';
 
@@ -7,6 +8,7 @@ describe('Feature: console-input', () => {
 
   beforeEach(() => {
     cy.login();
+    HomePagePo.goTo();
     chat.open();
     chat.isReady();
   });
