diff --git a/cypress/e2e/tests/features/console-input.spec.ts b/cypress/e2e/tests/features/console-input.spec.ts
index a6c4973..0525aea 100644
--- a/cypress/e2e/tests/features/console-input.spec.ts
+++ b/cypress/e2e/tests/features/console-input.spec.ts
@@ -44,7 +44,7 @@ describe('Feature: console-input', () => {
 
       console.autocompleteOverlay().should('be.visible');
 
-      cy.get('[data-testid="rancher-ai-ui-chat-input-textarea"]').type('{tab}', { force: true });
+      cy.get('[data-testid="rancher-ai-ui-chat-input-textarea"]').trigger('keydown', { key: 'Tab', keyCode: 9, which: 9, bubbles: true });
 
       console.autocompleteOverlay().should('not.exist');
       cy.get('[data-testid="rancher-ai-ui-chat-input-textarea"]')
